ui_src/src/components/radioButton/index.js

```diff
@@ -46,9 +46,33 @@ const RadioButton = (props) => {
                 defaultValue={radioValue || options[0]?.value}
             >
                 {options.map((option) => (
-                    <Radio key={option.id} value={option.value}>
-                        <span style={{ fontFamily: fontFamily }}>{option.label}</span>
-                    </Radio>
+                    <div
+                        style={{
+                            background: props.labelType && radioValue === option.value ? 'rgba(101, 87, 255, 0.1)' : '',
```

Comment:
Please try to extract the style to variable

Reply:
done